kernel/sys_clock.c

```diff
@@ -263,8 +263,8 @@ static inline void handle_timeouts(s32_t ticks)
 
 #ifdef CONFIG_TIMESLICING
 s32_t _time_slice_elapsed;
-s32_t _time_slice_duration = CONFIG_TIMESLICE_SIZE;
-int  _time_slice_prio_ceiling = CONFIG_TIMESLICE_PRIORITY;
+s32_t _time_slice_duration;
```

Comment:
duration and ceiling are now initialized to 0, right?

Reply:
Or, actually, they are initialized by `k_sched_time_slice_set` in` _sched_init`, right?

Reply:
Yes